src/cli/parse-ssl-options.js

```diff
@@ -0,0 +1,53 @@
+import fs from 'fs';
+import os from 'os';
+
+const MAX_PATH_LENGTH = os.type() === 'Windows_NT' ? 260 : Infinity;
```

Comment:
`MAX_PATH_LENGTH  === Infinity` means you can't pass a certificate content for properties that accept both file paths and file contents.